plugins/octant/go.sum

```diff
@@ -0,0 +1,683 @@
+cloud.google.com/go v0.26.0/go.mod h1:aQUYkXzVsufM+DwF1aE+0xfcU+56JwCaLick0ClmMTw=
```

Comment:
Has this file been tidy up? 
We should make `make tidy` and `make test-tidy` check it.

Reply:
Thanks for pointing out. I will update this part for tidying and checking both go.mod.